packages/logging/src/log.js

```diff
@@ -118,11 +118,10 @@ util.inherits(Log, commonGrpc.ServiceObject);
  */
 Log.assignSeverityToEntries_ = function(entries, severity) {
   return arrify(entries).map(function(entry) {
-    return extend(true, new Entry(), entry, {
-      metadata: {
-        severity: severity
-      }
-    });
+    var entry = extend(new Entry(), entry);
```

Comment:
Is removing `true` okay? I believe without it, modifying `entry.metadata` will affect the input `entry.metadata`.